pod/perlreref.pod

```diff
@@ -405,7 +405,7 @@ for details on regexes and internationalisation.
 =item *
 
 I<Mastering Regular Expressions> by Jeffrey Friedl
-(L<http://oreilly.com/catalog/9780596528126/>) for a thorough grounding and
+(L<https://oreilly.com/catalog/9780596528126/>) for a thorough grounding and
```

Comment:
This URL didn't resolve properly; suggest you use the same one you did earlier for Friedl's book.

Reply:
Thanks will amend